pages/index.js

```diff
@@ -1,7 +1,14 @@
+import Card from "@/components/Card";
+
 export default function HomePage() {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>Bingo</h1>
+      <h2>Play</h2>
+
+      <ul>
```

Comment:
when you use the ul in the component as suggested you don't need it here any longer.